### PR TITLE
Add .dumping command

### PR DESCRIPTION
**Pull request by JaffaCat**
This PR adds the .dumping command to Citra's Discord bot and also reorders most commands in alphabetical order. By default anyone can use this command to inform users on what the process of dumping a game is. 

_Suggested by SleepingSnake from the Citra Discord server._

### CODE REVIEW

File: citra.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      "log": { "reply": "For assistance with your issue, please post a log file. The following guide shows how to __get the log file__: <https://community.citra-emu.org/t/how-to-upload-the-log-file/296>"},
+      "modding": {"reply": "Please refer to our __game modding guide__: <https://citra-emu.org/help/feature/game-modding>"},
+      "multiplayer": {"reply": "To use Citra's multiplayer features, please refer to our __multiplayer setup guide__: <https://citra-emu.org/help/feature/multiplayer>"},
+      "portable": {"reply": "To download a portable build, go to <https://citra-emu.org/download/> and select `Manual download`. Then under **Nightly Build**, click on your operating system's icon to the right of the latest build available to download the build.\nExtract it (.7z can be extracted using Winrar or 7zip) and put it wherever you wish. Inside the extracted `nightly` folder, create a folder called `user`. This Citra should now store all of its config, save files and such inside of this `user` folder.\nCheck to make sure that this `user` folder is in the same folder that contains `citra-qt(.exe)`\nNow run the `citra-qt` executable found inside of the `nightly` folder."},
```
Comment (sakuramipha) [inline citra.json:25]: Why have the multiplayer and portable commands been changed? Please change those back to what they originally were, as the changes are unrelated to this PR.

File: citra.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
+{
```
Comment (sakuramipha) [inline citra.json:1]: This file should not be here. Please remove this. The commands file needs to be inside `src/responses`.

File: src/responses/citra.json [hunk 3]
```diff
@@ -1,36 +1,35 @@
...
-      "ios": {"reply": "Citra does not have an official iOS version because iOS is a complicated operating system to develop for. We currently do not have any developers with an interest in developing for this platform. There are some iOS forks of Citra, however they are not supported by us."},
+      "multiplayer": {"reply": "To use Citra's multiplayer features, please refer to our __multiplayer setup guide__: <https://citra-emu.org/help/feature/multiplayer>"},
+      "portable": {"reply": "To download a portable build, go to <https://citra-emu.org/download/> and select `Manual download`. Then under **Nightly Build**, click on your operating system's icon to the right of the latest build available to download the build.\nExtract it (.7z can be extracted using Winrar or 7zip) and put it wherever you wish. Inside the extracted `nightly` folder, create a folder called `user`. This Citra should now store all of its config, save files and such inside of this `user` folder.\nCheck to make sure that this `user` folder is in the same folder that contains `citra-qt(.exe)`\nNow run the `citra-qt` executable found inside of the `nightly` folder."},
```
Comment (sakuramipha) [inline src/responses/citra.json:25]: Please revert the changes on the multiplayer and portable commands. These do not need to be changed, and the changes are unrelated to this PR.

**Comment by FearlessTobi** [comment]
I don't see any good reason why the commands need to be reordered. 
I'd rather have a clean diff instead.
